pkg/utils/storage_unix.go

```diff
@@ -11,6 +11,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+//go:build !windows
```

Comment:
why add this

Reply:
this was auto added after `make fmt`( after updating my local go version to 1.17

Reply:
removed in https://github.com/pingcap/dm/pull/2121/commits/7e4c71574479ff2d2d96f675db0bb984c584fd32